doc/rules/class_notation/final_internal_class.rst

```diff
@@ -38,7 +38,7 @@ Default value: ``['@final', '@Entity', '@ORM\\Entity', '@ORM\\Mapping\\Entity',
 ``consider_absent_docblock_as_internal_class``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Should classes without any DocBlock be fixed to final?
+Whether classes without any DocBlock should be fixed to final.
```

Comment:
non blocking

I don't see issue with initial form, tbh.

Reply:
This was the only option in the form of question.